Modules/Bridge/VTK/include/itkVTKImageImport.h

```diff
@@ -30,7 +31,8 @@
       this->m_##name = _arg;                        \
       this->Modified();                             \
     }                                               \
-  }
+  }                                                 \
+  ITK_MACROEND_NOOP_STATEMENT
```

Comment:
Off-topic, but it looks like the macro that is fixed by this pull request (`itkSetMacro2`) isn't really necessary, is it? It's quite similar to the definition of `itkSetMacro`, at https://github.com/InsightSoftwareConsortium/ITK/blob/f9d5d97915bad80b95efacae3e8428352901247b/Modules/Core/Common/include/itkMacro.h#L1012-L1025